interface/billing/billing_process.php

```diff
@@ -46,6 +46,12 @@
   $BILLING_EXPORT = true;
 }
 
+// If UB-04 or Both selected in claim type include these files
+if ($GLOBALS['claim_type'] =='1' || $GLOBALS['claim_type'] =='2') {
+  include_once("$srcdir/gen_837i.inc.php");
```

Comment:
OK, yesterday...I think...I put that 1450 in the contribs repo...oh yeah, you got it.  I am so distracted and confused....